content/src/fieldConfig/profile.json

```diff
@@ -229,6 +238,11 @@
             "sole-proprietorship": "In a sole proprietorship you alone own the company and are responsible for its assets and liabilities. [Learn more about Sole Proprietorships.](https://business.nj.gov/pages/sole-proprietorship) "
           }
         },
+        "onboarding": {
+          "header": "Which Business Structure Will You Choose?",
+          "headerContextualInfo": "",
+          "description": "Your business’ `business structure|business-structure-learn-more` can have a large impact on taxes, financing, liability, and closure of your business. We strongly suggest doing research on the best business structure for your business.  \nSelect a business structure for your Navigator profile below. Making a selection will not form your business, it will help us create your personalized guide. You can change your intended business structure anytime before formally filing it with the NJ Department of Treasury."
+        },
```

Comment:
I think the overrides should the overrides be nested inside of onboarding? Otherwise they apply to Dakota businesses on the profile as well? 

(it would be true for oscar as well except we purposefully don't show legal structure on Oscar profile. But my point is that we are overriding the *onboarding* description so it feels like it should go inside the onboarding object?)

![Screenshot 2023-03-30 at 3 59 16 PM](https://user-images.githubusercontent.com/6563663/228950711-02a92711-d8cf-4148-9071-ed0b2801bf9e.png)

Reply:
Sure that makes sense

Reply:
So all 3 onboarding flows should be correct now. And all 3 profiles should have all the contextual headers from design (with a few minor inconsistencies)